docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -58,6 +58,8 @@ Access
 
 - The ability to create issues in `the ClusterHQ JIRA <https://clusterhq.atlassian.net/secure/Dashboard.jspa>`_.
 
+- Access to ClusterHQ's Google Drive.
```

Comment:
This should have some indication of its purpose, so that someone outside ClusterHQ wishing to follow the release process (for some internal purpose) would know whether this is a problem for them following these instructions.

e.g. "...for access to ClusterHQ versioning policy documents."